server/src/main/java/org/apache/druid/server/emitter/EmitterModule.java

```diff
@@ -82,33 +83,49 @@ public void configure(Binder binder)
 
     binder.bind(Emitter.class).toProvider(new EmitterProvider(emitterType)).in(LazySingleton.class);
 
-    MapBinder<String, String> extraServiceDimensions = MapBinder.newMapBinder(
+    MapBinder<String, String> extraServiceDimensionsLegacy = MapBinder.newMapBinder(
```

Comment:
maybe requiredExtraServiceDimension or leave as is and other one is optionalExtraServiceDimensions...